cirq/linalg/decompositions.py

```diff
@@ -168,10 +149,8 @@ def map_eigenvalues(
     Returns:
         The transformed matrix.
     """
-    vals, vecs = _perp_eigendecompose(matrix,
-                                      rtol=rtol,
-                                      atol=atol)
-    pieces = [np.outer(vec, np.conj(vec.T)) for vec in vecs]
+    vals, vecs = (matrix, atol=atol)
```

Comment:
You probably meant

```python
vals, vecs = unitary_eig(matrix, atol=atol)
```

Reply:
Yes that was an accidental mouse drag:) fixed it!